{"metadata":{"roleHeading":"API Collection","title":"Auth0Error Implementations","role":"collectionGroup","modules":[{"name":"Auth0"}]},"identifier":{"interfaceLanguage":"swift","url":"doc:\/\/com.auth0.Auth0\/documentation\/Auth0\/CredentialsManagerError\/Auth0Error-Implementations"},"topicSections":[{"title":"Instance Properties","generated":true,"identifiers":["doc:\/\/com.auth0.Auth0\/documentation\/Auth0\/CredentialsManagerError\/errorDescription","doc:\/\/com.auth0.Auth0\/documentation\/Auth0\/CredentialsManagerError\/localizedDescription"]}],"kind":"article","hierarchy":{"paths":[["doc:\/\/com.auth0.Auth0\/documentation\/Auth0","doc:\/\/com.auth0.Auth0\/documentation\/Auth0\/CredentialsManagerError"]]},"schemaVersion":{"major":0,"minor":3,"patch":0},"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/auth0\/credentialsmanagererror\/auth0error-implementations"]}],"sections":[],"references":{"doc://com.auth0.Auth0/documentation/Auth0":{"role":"collection","type":"topic","abstract":[{"type":"text","text":"SDK for Apple platforms."}],"kind":"symbol","identifier":"doc:\/\/com.auth0.Auth0\/documentation\/Auth0","url":"\/documentation\/auth0","title":"Auth0"},"doc://com.auth0.Auth0/documentation/Auth0/CredentialsManagerError":{"kind":"symbol","identifier":"doc:\/\/com.auth0.Auth0\/documentation\/Auth0\/CredentialsManagerError","fragments":[{"text":"struct","kind":"keyword"},{"kind":"text","text":" "},{"text":"CredentialsManagerError","kind":"identifier"}],"navigatorTitle":[{"text":"CredentialsManagerError","kind":"identifier"}],"abstract":[{"type":"text","text":"Represents an error during a Credentials Manager operation."}],"url":"\/documentation\/auth0\/credentialsmanagererror","title":"CredentialsManagerError","role":"symbol","type":"topic"},"doc://com.auth0.Auth0/documentation/Auth0/CredentialsManagerError/localizedDescription":{"role":"symbol","fragments":[{"kind":"keyword","text":"var"},{"text":" ","kind":"text"},{"text":"localizedDescription","kind":"identifier"},{"kind":"text","text":": "},{"text":"String","preciseIdentifier":"s:SS","kind":"typeIdentifier"}],"url":"\/documentation\/auth0\/credentialsmanagererror\/localizeddescription","type":"topic","abstract":[{"text":"Description of the error.","type":"text"}],"kind":"symbol","title":"localizedDescription","identifier":"doc:\/\/com.auth0.Auth0\/documentation\/Auth0\/CredentialsManagerError\/localizedDescription"},"doc://com.auth0.Auth0/documentation/Auth0/CredentialsManagerError/errorDescription":{"url":"\/documentation\/auth0\/credentialsmanagererror\/errordescription","identifier":"doc:\/\/com.auth0.Auth0\/documentation\/Auth0\/CredentialsManagerError\/errorDescription","type":"topic","fragments":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"errorDescription"},{"text":": ","kind":"text"},{"kind":"typeIdentifier","preciseIdentifier":"s:SS","text":"String"},{"kind":"text","text":"?"}],"kind":"symbol","role":"symbol","abstract":[{"text":"Description of the error.","type":"text"}],"title":"errorDescription"}}}